src/frontend/src/tests/components/CheckList.test.tsx

```diff
@@ -13,6 +13,8 @@ import * as descBulletHooks from '../../hooks/description-bullets.hooks';
 import { mockAuth } from '../test-support/test-data/test-utils.stub';
 import { exampleAdminUser } from '../test-support/test-data/users.stub';
 import { mockCheckDescBulletReturnValue } from '../test-support/mock-hooks';
+import FormCheckInput from 'react-bootstrap/esm/FormCheckInput';
+import { check } from 'prettier';
```

Comment:
Lint check doesn't like likes 16 and 17, you never used these imports so remove these 2 lines